packages/frontend/src/features/catalog/components/CatalogPanel.tsx

```diff
@@ -296,6 +312,10 @@ export const CatalogPanel: FC = () => {
                 field: selectedItem.field,
             });
 
+            if (selectedItem.group === TABLES_WITH_ERRORS_GROUP_NAME) {
+                setSidebarOpen(false);
+                return; // no metadata for tables with errors
+            }
```

Comment:
should we move this to the line before ?
```
if (!isSidebarOpen) {
                setSidebarOpen(true);
            }
```

Reply:
I see, good idea